src/ucs/debug/log.c

```diff
@@ -145,15 +145,24 @@ static void ucs_log_get_file_name(char *log_file_name, size_t max, int idx)
 
 static void ucs_log_file_rotate()
 {
-    char old_log_file_name[PATH_MAX];
-    char new_log_file_name[PATH_MAX];
+    char *old_log_file_name = NULL;
+    char *new_log_file_name = NULL;
     int idx, ret;
 
+    old_log_file_name = (char *)ucs_malloc(PATH_MAX, "old_log_file_name");
```

Comment:
in C this cast from `void *` is redundant

Reply:
done